Artesian/Artesian.SDK/Dto/DerivedCfg/DerivedCfgBase.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) ARK LTD. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for
+// license information. 
+using MessagePack;
+using Newtonsoft.Json;
+
+namespace Artesian.SDK.Dto
+{
+    /// <summary>
+    /// Derived Curve Configuartion
+    /// </summary>
+    [MessagePackObject]
+    [Union(0, typeof(DerivedCfgMuv))]
+    [Union(1, typeof(DerivedCfgCoalesce))]
+    [JsonConverter(typeof(DerivedCfgBaseConverter))]
```

Comment:
Artesian.SDK either use STJ or Newtonsoft, for sure not both. I don't remember which one but do not introduce a new dependency.